specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/serviceEndpointPolicy.json

```diff
@@ -0,0 +1,634 @@
+      },
+      "patch": {
+        "tags": [
+          "ServiceEndpointPolicies"
+        ],
+        "operationId": "ServiceEndpointPolicies_UpdateTags",
+        "description": "Updates service Endpoint Policies tags.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "ServiceEndpointPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the service endpoint policy."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./network.json#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to update service endpoint policy tags."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting ServiceEndpointPolicy resource.",
+            "schema": {
+              "$ref": "#/definitions/ServiceEndpointPolicy"
+            }
+          }
+        },
+        "x-ms-examples": {
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/ServiceEndpointPolicies": {
+      "get": {
+        "tags": [
+          "ServiceEndpointPolicies"
+        ],
+        "operationId": "ServiceEndpointPolicies_ListAll",
+        "description": "Gets all the service endpoint policies in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ServiceEndpointPolicy resources.",
+            "schema": {
+              "$ref": "#/definitions/ServiceEndpointPolicyListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all network security groups": { "$ref": "./examples/ServiceEndpointPolicyListAll.json" }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/serviceEndpointPolicies": {
+      "get": {
+        "tags": [
+          "serviceEndpointPolicies"
+        ],
+        "operationId": "serviceEndpointPolicies_List",
```

Comment:
This should be called "ListByResourceGroup"

Reply:
Fixed